.projenrc.ts

```diff
@@ -33,6 +33,7 @@ const project = new awscdk.AwsCdkConstructLibrary({
     distName: 'open-constructs-aws-cdk',
     module: 'open_constructs_aws_cdk',
   },
+  minNodeVersion: '18.20.2',
```

Comment:
This seems overly prescriptive. Why not this? 

```suggestion
  minNodeVersion: '18.0.0',
```

Reply:
Makes sense!

Thanks.